.azure/pipelines/pr/Phonebook.Backend.PictureService.pr.yml

```diff
@@ -0,0 +1,16 @@
+# Starter pipeline
```

Comment:
Great! Can you add Hooks and Trigger Paths to it aswell? Just have a look here: https://github.com/T-Systems-MMS/phonebook/blob/master/.azure/pipelines/pr/Phonebook.Frontend.pr.yml

Reply:
Done.